src/main/scala/cromwell/engine/io/IOManager.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.engine.io
+
+import cromwell.binding.IoInterface
+
+//TODO: Use reflection to automagically implement all IOInterface methods ?
```

Comment:
I noticed the words reflection and macros when scanning my email this morning. If either of those end up being the final proposed solution, let's chat :)